README.md

```diff
@@ -41,102 +41,100 @@ Known Issues
 | masters.instances | Master instances IDs |
-| masters.os_user | Master instances private OS default user |
+| masters.os\_user | Master instances private OS default user |
 | masters.prereq-id | Returns the ID of the prereq script for masters (if user_data or ami are not used) |
-| masters.private_ips | Master instances private IPs |
-| masters.public_ips | Master instances public IPs |
-| private_agents.instances | Private Agent instances IDs |
-| private_agents.os_user | Private Agent instances private OS default user |
-| private_agents.prereq-id | Returns the ID of the prereq script for private agents (if user_data or ami are not used) |
-| private_agents.private_ips | Private Agent instances private IPs |
-| private_agents.public_ips | Private Agent public IPs |
-| public_agents.instances | Private Agent |
-| public_agents.os_user | Private Agent instances private OS default user |
-| public_agents.prereq-id | Returns the ID of the prereq script for public agents (if user_data or ami are not used) |
-| public_agents.private_ips | Public Agent instances private IPs |
-| public_agents.public_ips | Public Agent public IPs |
-| security_groups.admin | This is the id of the admin security_group that the cluster is in |
-| security_groups.internal | This is the id of the internal security_group that the cluster is in |
-| vpc.cidr_block | This is the cidr_block of the VPC the cluster is in |
+| masters.private\_ips | Master instances private IPs |
+| masters.public\_ips | Master instances public IPs |
+| private\_agents.instances | Private Agent instances IDs |
+| private\_agents.os\_user | Private Agent instances private OS default user |
+| private\_agents.prereq-id | Returns the ID of the prereq script for private agents (if user_data or ami are not used) |
+| private\_agents.private\_ips | Private Agent instances private IPs |
+| private\_agents.public\_ips | Private Agent public IPs |
+| public\_agents.instances | Public Agent instances IDs |
+| public\_agents.os\_user | Private Agent instances private OS default user |
+| public\_agents.prereq-id | Returns the ID of the prereq script for public agents (if user_data or ami are not used) |
+| public\_agents.private\_ips | Public Agent instances private IPs |
+| public\_agents.public\_ips | Public Agent public IPs |
+| security\_groups.admin | This is the id of the admin security_group that the cluster is in |
+| security\_groups.internal | This is the id of the internal security_group that the cluster is in |
+| vpc.cidr\_block | This is the cidr_block of the VPC the cluster is in |
 | vpc.id | This is the id of the VPC the cluster is in |
-| vpc.main_route_table_id | This is the id of the VPC's main routing table the cluster is in |
-| vpc.subnet_ids | This is the list of subnet_ids the cluster is in |
+| vpc.main\_route\_table\_id | This is the id of the VPC's main routing table the cluster is in |
+| vpc.subnet\_ids | This is the list of subnet_ids the cluster is in |
```

Comment:
It looks like this recent commit is attempting to escape underscores

Reply:
We have moved to latest version of terraform-docs for this release per @fatz

Reply:
@bernadinm y which makes sense as per markdown you change _textstyle_ with \_

Reply:
terraform-docs in this version is also already like that integrated into the terraform-jenkins container image ;)
so the CI will use the same for testing and later creation of README.md ;)